dirty.rb

```diff
@@ -0,0 +1,5 @@
+begin
+  1
+rescue
```

Comment:
__[development][rubocop]__
* Do not suppress exceptions. [:ref](https://github.com/bbatsov/ruby-style-guide#dont-hide-exceptions)